src/Sticky.jsx

```diff
@@ -283,7 +283,21 @@ class Sticky extends Component {
         this.delta = delta;
     }
 
+    updateElements ( nextProps ) {
```

Comment:
Hate to be that guy, but the spacing is different to the rest of the code, IMO it's better to keep it consistent with however the rest of the project's code has it